pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -607,6 +607,11 @@ type serviceLoader struct {
 	items     []kapi.Service
 }
 
+// NewServiceLoader returns a new serviceLoader
+func NewServiceLoader(namespace string, lister kclient.ServicesNamespacer) *serviceLoader {
```

Comment:
I've removed those funcs. We can add them later if we find any use for them elsewhere.